crest/Assets/Crest/Crest/Scripts/Shapes/FFT/ShapeFFT.cs

```diff
@@ -527,21 +558,28 @@ void BakeHelpBox(ShapeFFT target)
 
         public override void OnInspectorGUI()
         {
-            base.OnInspectorGUI();
+            var target = this.target as ShapeFFT;
 
-            var fft = target as ShapeFFT;
+            if (target._isPrefabStageInstance)
+            {
+                EditorGUILayout.Space();
+                EditorGUILayout.HelpBox("Crest does not support prefab mode. Changes made in prefab mode will not be reflected in the scene view until saved.", MessageType.Warning);
```

Comment:
similar comment - directive "save the prefab to see changes" woudl get my vote

Reply:
Done.